reactivesocket-core/build.gradle

```diff
@@ -14,30 +14,24 @@
  * limitations under the License.
  */
 
-buildscript {
-    repositories {
-        maven { url "https://plugins.gradle.org/m2/" }
-    }
-
-    dependencies {
-        classpath 'gradle.plugin.me.champeau.gradle:jmh-gradle-plugin:0.3.0'
-    }
+plugins {
+    id "me.champeau.gradle.jmh" version "0.3.1"
 }
 
-apply plugin: 'me.champeau.gradle.jmh'
-
 jmh {
     jmhVersion = '1.15'
-    jvmArgs = '-XX:+UnlockCommercialFeatures -XX:+FlightRecorder'
+    jvmArgs = '-XX:+UseG1GC -Xmx16g -Xms16g -XX:+UnlockCommercialFeatures -XX:+FlightRecorder'
     profilers = ['gc']
     zip64 = true
     warmupBatchSize = 10
     iterations = 500
-
+    duplicateClassesStrategy = 'warn'
 }
 
 dependencies {
-    compile project(':reactivesocket-publishers')
+    compile 'io.projectreactor:reactor-core:3.0.5.RELEASE'
```

Comment:
Should we shade this and only expose the ReactiveStreams interfaces? Or are we okay with this public dependency?

The problem would come if Reactor makes breaking changes, when it should just be an implementation detail.